test/conformance/testing/include/uur/checks.h

```diff
@@ -69,4 +110,15 @@ inline std::ostream &operator<<(std::ostream &out,
   return out;
 }
 
+#ifndef ASSERT_GET_INFO
+#define ASSERT_GET_INFO(CALL, TYPE)                                            \
+  do {                                                                         \
+    ASSERT_SUCCESS(CALL);                                                      \
+    TYPE returned_value = property_value;                                      \
```

Comment:
Not sure about the assumption that we're going to have a variable called `property_value` declared before using the macro. Could we make it take CALL, PROPERTY_VALUE and then this line looks like
```suggestion
    auto returned_value = PROPERTY_VALUE;                                      \
```
I also would prefer a more specific name, maybe like `ASSERT_QUERY_RETURNS_VALUE`

Reply:
that makes sense thanks, I've changed the macro over now